app/config/image_variations.yml

```diff
@@ -20,3 +20,9 @@ ezpublish:
                     reference: ~
                     filters:
                         - { name: geometry/scalewidthdownonly, params: [710] }
+liip_imagine:
+    driver: gd
+    filter_sets:
+        blog_thumbnail:
```

Comment:
Maybe I'm wrong, but it seems `post-processors` works in the same way, I mean that we have to apply `post-processor` to each filter separately as well.